tests/pools/test_pool_wallet.py

```diff
@@ -97,7 +97,7 @@ async def mock_get_current_state(self: Any) -> Any:
 
     # Create an empty PoolWallet and populate only the required fields
     wallet = PoolWallet(
-        wallet_state_manager=MockWalletStateManager(),
+        wallet_state_manager=MockWalletStateManager(),  # type: ignore
```

Comment:
When ignoring errors, ignore a specific error.  `# type: ignore[param-type]` or whatever it is.  Also, a comment is generally warranted.  But...  I think in this case it's obvious and a cast would be the most specific thing to do.

```suggestion
        wallet_state_manager=cast(WalletStateManager, MockWalletStateManager()),
```